src/lang/cs_CZ.ts

```diff
@@ -353,15 +353,15 @@ export default {
   "Image": "Snímek",
   "Data": "Data",
   "Print": "Tisk",
-  "Click, tap or press ENTER to open": "",
-  "Click, tap or press ENTER to print.": "",
-  "Click, tap or press ENTER to export as %1.": "",
-  'To save the image, right-click this link and choose "Save picture as..."': "",
-  'To save the image, right-click thumbnail on the left and choose "Save picture as..."': "",
-  "(Press ESC to close this message)": "",
-  "Image Export Complete": "",
-  "Export operation took longer than expected. Something might have gone wrong.": "",
-  "Saved from": "",
+  "Click, tap or press ENTER to open": "Klikněte nebo stiskněte ENTER pro otevření",
+  "Click, tap or press ENTER to print.": "Klikněte nebo stiskněte ENTER pro tisk",
+  "Click, tap or press ENTER to export as %1.": "Klikněte nebo stiskněte ENTER pro uložení jako %1",
+  'To save the image, right-click this link and choose "Save picture as..."': 'Pro uložení obrázku klikněte na odkaz pravým tlačítkem a zvolte "Uložit obrázek jako..."',

```

Comment:
Czech lower and upper quotation marks `„“` should be used instead of english ones `""`. Here and in other places.